### ISSUE TITLE
Notification for on_created not triggering always

### ISSUE THREAD

**Issue by redfellow** [Tautulli/Tautulli#1182]
**Version:** V2.0.1-BETA
**Git Branch:** master
**Git Commit Hash:** cfd30996264b7e9fe4ef87f02d1cc52d1ae8bfca
**Operating system:** Windows 10
**Python version:** Python 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 20:40:30) [MSC v.1500 64 bit (AMD64)] on win32
**What you did?** Dropped few new folders to library
**What happened?** Plex triggered partial scan automaticly. New media shows up on recently added in Plex and Tautulli, but no notification was pushed or even logged as pushed
**What you expected?** Tautulli to push a `on_created` notification as usually
**Link to logs:** https://vgy.me/bGnL56.png

**Comment by JonnyWong16** [comment]
Please provide correct info. You cannot be on the beta version on the master branch and master commit. Also, the beta is currently on version v2.0.4-beta.

**Comment by redfellow** [comment]
Well, I just gave the info as reported by Tautulli itself: https://vgy.me/tgff42.png

Hitting Check for updates just reloads the front page of Tautulli. Maybe my installation is fubar somehow? I went and checked out beta branch now.

Edit: now it says beta branch, but hitting "Check for updates" still navigates to the front page after a brief delay and the version stays at `v2.0.1-BETA`. 

Edit2: Realized it was working behind the scenes, got the update popup in the lower corner now finally. Now my Tautulli is up to date and I'll reopen this ticket if I can repeat the issue.
